--- /etc/pam.d/login.orig
+++ /etc/pam.d/login
@@ -2,7 +2,9 @@
 auth       required   pam_securetty.so
 auth       requisite  pam_nologin.so
 auth       include    system-local-login
+auth       optional   pam_kwallet5.so
 account    required   pam_nologin.so
 account    include    system-local-login
 password   include    system-local-login
 session    include    system-local-login
+session    optional   pam_kwallet5.so force_run

